AutoCollection/diagnostic-channel/mongodb.sub.ts

```diff
@@ -36,4 +41,9 @@ export function enable(enabled: boolean, client: TelemetryClient) {
             channel.unsubscribe("mongodb", subscriber);
         }
     }
-}
\ No newline at end of file
+}
+
+export function dispose() {

```

Comment:
Where this will be called?

Reply:
It's currently a convenience function for tests only since `clients` is not exported. Perhaps instead I should just export `clients` and implement `dispose()` in tests?